docs/DeveloperGuide.md

```diff
@@ -308,12 +308,22 @@ Priorities: High - must have; Medium - nice to have;  Low - unlikely to have.
+| L3  | `HIGH`   | user                                       | add the corresponding subject to my lessons                         | plan and prepare for that specific subject                              |
+| L4  | `HIGH`   | user                                       | add each piece of homework I gave to my students for the lesson     | keep track of what I have to go through that lesson                     |
+| L5  | `HIGH`   | user                                       | delete a student's lesson                                           | remove lessons that I no longer have                                    |
+| L6  | `MEDIUM` | user                                       | edit a student's recurring lesson's date                            | update my lesson schedule in case of changes                            |
+| L7  | `MEDIUM` | user                                       | edit a student's lesson time                                        | update my lesson schedule in case of changes                            |
+| L8  | `MEDIUM` | user                                       | edit a student's lesson subject                                     | change the subject field in case of changes                             |
+| L9  | `MEDIUM` | user                                       | edit a student's homework for that particular lesson                |                                                                         |
+| L10 | `MEDIUM` | user                                       | add makeup lessons with specific time and date for a student        | track lessons outside of my usual lesson schedule                       |
+| L11 | `MEDIUM` | user                                       | add homework for the makeup lessons                                 | track lessons outside of my usual lesson schedule                       |
+| L12 | `MEDIUM` | user                                       | add subject for the makeup lessons                                  | track lessons outside of my usual lesson schedule                       |
+| L13 | `MEDIUM` | user                                       | view the lessons I have for the week                                | easily manage my schedule                                               |
+| L15 | `MEDIUM` | user                                       | view the homework I have set for my student after the lesson        | prepare for my upcoming lessons                                         |
+| L16 | `MEDIUM` | user                                       | view the subject of my upcoming lessons                             | prepare for my upcoming lessons                                         |
+| L17 | `MEDIUM` | user                                       | view a calendar of my scheduled lessons                             | plan ahead while managing my schedule                                   |
```

Comment:
whats the diff between L13 (view lessons for the week) and L17 (view calendar)?

Reply:
this user story was there before I changed it, I just changed the numbering. Perhaps @Chesterwongz can clarify since you are doing schedule feature?

Reply:
looks the same, ill remove it